### DISCUSSION TITLE
How to change .tab-lifted.tab-active's background-color, using tailwind-css classes?

### DISCUSSION THREAD

**Discussion by cmal** [saadeghi/daisyui#2312] [Q&A] [answered]
How to change .tab-lifted.tab-active's background-color, using tailwind-css classes?

**Comment by saadeghi** [comment, marked as answer]
There's an internal CSS variable `--tab-bg` for that. It handles the background color and also the rounded corners.  
You can change it with a class name like `[--tab-bg:red]`

https://play.tailwindcss.com/9rQQ92sr1X

**Reply by Ayabara1013** [reply]
how would I change the border colour?

**Reply by saadeghi** [reply]
@Ayabara1013 `[--tab-border-color:red]`  
https://play.tailwindcss.com/3RbtHeNIOz
